components/com_kunena/site/language/en-GB/en-GB.com_kunena.views.ini

```diff
@@ -125,6 +125,8 @@ COM_KUNENA_SELECT_CATEGORY="- Select Category -"
 COM_KUNENA_VIEW_DISABLED="The administrator has disabled public write access."
 COM_KUNENA_VIEW_TOPICS_DEFAULT="Topic: %s"
 COM_KUNENA_VIEW_TOPIC_ERROR_LOOP="Internal Error: Loop detected in moved topics!"
+COM_KUNENA_LOGIN_PLEASE="Please %s %s to join the conversation."
+COM_KUNENA_LOGIN_OR="or"
```

Comment:
Splitting sentences like this doesn't really work in some languages.

    COM_KUNENA_LOGIN_PLEASE="Please %s to join the conversation."
    COM_KUNENA_LOGIN_X_OR_Y="%s or %s"

Might be better?